easycv/models/backbones/repvgg_yolox_backbone.py

```diff
@@ -0,0 +1,393 @@
+# borrow some code from https://github.com/DingXiaoH/RepVGG/repvgg.py MIT2.0
```

Comment:
rename repvgg_yolox

Reply:
It's an efficient backbone of YOLOX based on RepVGG, so we name the backbone as RepVGGYOLOX.